### PR TITLE
Syndicate Rebalancing + Gun rebalancing

### PR DESCRIPTION
**Pull request by TooSillyFennec**
## About the PR
Major price changes on underused items and some general rebalance for synd gear and guns

## Why / Balance

only the gear changes are mentioned here. Price changes can be discussed if there are any changes/reasons wanted.

Cane Blade Reworked - The Cane blade's now 40 TC, but a sidegrade to e-sword. Less than an e-sword DPS at 24 DPS, but 1.5 hit speed and now ignites as a reference to the fact SS13 curators are related to wizards in some ways. Current Cane Blade was well pretty... bad and overpriced.

Synd EVA Reworked - Synd EVA now is closer to the SS13 Syndicate spacesuit- AKA a low defense EVA suit. also made it have 0 slow. Cost changed to 20 TC. - Why? Because you can get the same thing for 0 TC without the valid when it has the same stats as the normal EVA, and having a lighter EVA yet still armored option rounds things out.  25% Combat types means the suit gives 1-2 hits of extra survivability. 50% Rads to add an extra reason to buy one, aka the goose is loose :3

Fake insuls buff - By Buff I mean insta-crit. The effort required to 1: Shock someone w/ them 2: have them be alone or no non-antags are w/ them and 3: Hide the body before they can be found is insane. Also very target dependant and may need thief gloves, so 10/30 TC for a niche situational kill needs to well... actually work! 

China Lake Nerf - 1.0 FR > 0.6 FR. it's a grenade launcher that's an aoe insta death. 0.6 FR makes shots need a lot more thought. The skill weapon needs actual skill not just "Click pixel this far away, girl!!!"

Hristov Changes - Hristovs now deal  deal 60 AP Damage and 90 Structural. They also have 6 Ammo Capacity. Price has been increased from 60 TC to 80 TC. Boxes now have 12 .60 and are 40 TC. FR is actually 0.8 instead of 0.75. This makes them an actual weapon.

Revolver Changes - All normal Revolvers except mateba have recieved a 0.5 FR increase. AKA - inspector 2.0, Deckard/Python 2.5.

.35 Pistol Buff - 10 > 12 Capacity. Pistols have more ammo than you'd think, and this makes them somewhat closer to 13 pistol balance, but not the exact 2 kills for every normal mag the synd makarov if every shot landed on 13.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Clearance Sale! A high amount of items that haven't sold well have been made cheaper by the syndicate. However, Suppliers have increased prices on certain items.
- tweak: The Cane blade has been totally rebalanced, dealing more damage and igniting targets, but has been marked up in price to 40 TC.
- tweak: The old Syndicate EVA Suit has been replaced by a new lightweight durathread model, with some moderate protection and anti-rad but costs 20 TC. Reviewers have stated that it's insanely comfy, but these have been bought, NT claims.
- tweak: The China Lakes in supply  have started to deprecate and fire slower due to age. 1 FR > 0.6 FR.
- tweak: The Syndicate has realised they were sending .40 Materiel instead of .60 Antimateriel. This has been rectified recently and the hristov improved. The correct ammo is very costly! 40 Damage > 60 Damage AP, Hristov 60 TC > 80 TC.
- tweak: Revolvers across the galaxy have had their firerates magically increased after one girl decided "Hey, maybe we should invest more money into the revolver department!" 1.5 > 2.0 FR for Inspector, 2.0 > 2.5 FR for Deckard and Python.
- tweak: .35 Auto Pistol Capacity 10 > 12
- tweak: Slug Shells now deal 40 Damage making them a more desirable option!

### CODE REVIEW

**Comment by CatMagic1** [comment]
This is all great.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -135,7 +135,7 @@
   productEntity: BriefcaseSyndieSniperBundleFilled
   cost:
     # Goobstation - Telecrystal rework
-    Telecrystal: 60
+    Telecrystal: 110 # increase price by a lot when the thing has a scope
```
Comment (Piras314) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:138]: It has a scope now???????

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 2]
```diff
@@ -482,7 +482,7 @@
   productEntity: MagazineBoxAntiMateriel
   cost:
     # Goobstation - Telecrystal rework
-    Telecrystal: 10
+    Telecrystal: 40 # 2 ammo capacity worth
```
Comment (Piras314) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:485]: Why are you nerfing the worst and least used weapon in the game, the sniper

**Comment by lanse12** [comment]
how could you do this to me... nerfing my china lake...

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 3]
```diff
@@ -1143,8 +1143,8 @@
   productEntity: ScramImplanter
   cost:
     # Goobstation - Telecrystal rework
-    Telecrystal: 30 # it's a gamble that may kill you easily so 30 TC per 2 uses, second one more of a backup
-  categories:
+    Telecrystal: 20 # it's a gamble that may kill you easily but can also save an agent. and that's why the 20 TC Balance tier makes sense.
+  catergories:
```
Comment (Piras314) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1147]: categories, not catergories

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 4]
```diff
@@ -1575,7 +1575,7 @@
   productEntity: ClothingBackpackDuffelSyndicateCarpSuit
   cost:
     # Goobstation - Telecrystal rework
-    Telecrystal: 20
+    Telecrystal: 8 # Why ever buy a 20 TC fake over a 20 TC real deal?
```
Comment (Piras314) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1578]: The carp suit works as a hardsuit that you can eat and drink and everything like that out of, it's actually really good, put this back to 20 or at least 15

File: Resources/Prototypes/Entities/Clothing/Hands/colored.yml [hunk 5]
```diff
@@ -350,6 +350,6 @@
   suffix: Conducting
   components:
   - type: Insulated
-    coefficient: 5 # zap em good
+    coefficient: 10 # if you pull a shock off and they are alone and can never be found, a kill's deserved. *BZZT!!!* - Goobstation
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Clothing/Hands/colored.yml:353]: no

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml [hunk 6]
```diff
@@ -5,9 +5,8 @@
...
   components:
   - type: Projectile
-    damage:
+    damage: # Goobstation
       types:
-        Piercing: 40
-        Structural: 30
-  - type: StaminaDamageOnCollide
-    damage: 35
+        Piercing: 60
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml:10]: structural damage doesn't actually work on people other than like, borgs right? if so this is fine, even at the unmodified price

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml [hunk 7]
```diff
@@ -10,7 +10,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 28
+        Piercing: 45 # Goobstation - Don't really have to cycle them here, so a moderate nerf over 13 makes sense... not 28 damage what the actual fuck was that?
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml:13]: 40 or 35

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 8]
```diff
@@ -115,7 +115,7 @@
     sprite: Objects/Weapons/Guns/Shotguns/db_shotgun_inhands_64x.rsi
   - type: GunRequiresWield #remove when inaccuracy on spreads is fixed
   - type: Gun
-    fireRate: 2
+    fireRate: 4 # until cycling works, this works to simulate the fact a DB doesn't need to pump- and also removes sawn off powergame yayyyy :3 - Goobstation
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:118]: sawn off SHOULD be faster

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 9]
```diff
@@ -173,7 +173,7 @@
     sprite: Objects/Weapons/Guns/Shotguns/pump.rsi
   - type: GunRequiresWield #remove when inaccuracy on spreads is fixed
   - type: BallisticAmmoProvider
-    capacity: 4
+    capacity: 6 # Goobstation
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:176]: hell no

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 2 above)
Comment (Piras314) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:485]: this is actually fair

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 1 above)
Comment (Piras314) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:138]: 12 is fine even with the buff

Comment (TooSillyFennec) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:138]: the ammo does 60 AP now- that was a comment for when the thing has one, not what's in the PR.

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 2 above)
Comment (TooSillyFennec) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:485]: The ammo's buffed, and that's 2 capacity worth

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 3 above)
Comment (TooSillyFennec) [reply Resources/Prototypes/Catalog/uplink_catalog.yml]: yeah, fixed that, me made oopsie

**Comment by TooSillyFennec** [comment]
fuck damn it accidentally made a review :/

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 1 above)
Comment (Piras314) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:138]: 110 is absolutely insane still, agents start with 100 and should be able to buy it, with its low fire rate and high cost of ammo

Comment (Piras314) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:138]: 80tc maybe

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 4 above)
Comment (Piras314) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:1578]: @TooSillyFennec

**Review by Piras314 — APPROVED** [review]

**Comment by Piras314** [comment]
No merge for a few days for player / contributor feedback

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml [hunk 10]
```diff
@@ -51,6 +51,8 @@
     tags:
       - CaneBlade
   - type: DisarmMalus
+  - type: IgniteOnMeleeHit # Goobstation
+    fireStacks: 1
```
Comment (misandrie) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:55]: Congratulations you have created a portable incinerator

File: Resources/Prototypes/Goobstation/Entities/Clothing/OuterClothing/softsuits.yml [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+  - type: ClothingSpeedModifier
+    walkModifier: 1.0
+    sprintModifier: 1.0
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.75
+        Slash: 0.75
+        Piercing: 0.75
+        Radiation: 0.5
```
Comment (misandrie) [inline Resources/Prototypes/Goobstation/Entities/Clothing/OuterClothing/softsuits.yml:21]: >25% flat resists on brute plus 50% res on rads for zero movement penalty

Comment (Piras314) [reply Resources/Prototypes/Goobstation/Entities/Clothing/OuterClothing/softsuits.yml:21]: ZERO MOVEMENT PENALTY? oh hell no I didn't realise, this should have like 5% resists then, 10 max

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 4 above)
Comment (lanse12) [reply Resources/Prototypes/Catalog/uplink_catalog.yml]: there's two one is fake one is real

File: Resources/Prototypes/Goobstation/Entities/Clothing/OuterClothing/softsuits.yml (hunk 11 above)
Comment (misandrie) [reply Resources/Prototypes/Goobstation/Entities/Clothing/OuterClothing/softsuits.yml:21]: Piras314 approved these changes 2 hours ago

**Comment by Aidenkrz** [comment]
Genuinely one of the only good balance PRs that's been made here, handle the two reviews above and this looks good.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml (hunk 10 above)
Comment (Piras314) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:55]: yeah it should be lowered if possible

**Comment by Verbalase** [comment]
chat there might be a decent pr (no idea)

**Comment by JoeHammad1844** [comment]
that penetration mechanic that was denied in wizden would be cool. where the hristov could shoot through walls

**Comment by TooSillyFennec** [comment]
I fixed a recent den fuck up that made tots have less objectives. old tot objs always overshot to above 5, rarely above 6 diff and they removed that. well that causes 2 objective tots. Best way to emulate the weight overshoot? increase the weight to where it would overshoot, 6.

Why did den not do this? _We all know why._

**Comment by misandrie** [comment]
Diff 6 usually ended up in objectives which were tantamount to murdering the entire station.

It is been capped for a reason.

**Comment by TooSillyFennec** [comment]
> Diff 6 usually ended up in objectives which were tantamount to murdering the entire station.
> 
> It is been capped for a reason.

Current Diff is 5, and den fixed the overflow that caused that. overflow actually caused synds to have 6 diff pretty regularly on 5. Basically, keeps the overflow fix but makes objs roll closer to how they used to.

**Comment by TooSillyFennec** [comment]
objs

There's an example. Also many cases of obj sets that can't be gotten anymore, like 2 steal objectives.

**Comment by misandrie** [comment]
Steal objectives are more often than not kill objectives with extra steps due to average command staff behavior and how the game functions in general.

**Comment by TooSillyFennec** [comment]
> Steal objectives are more often than not kill objectives with extra steps due to average command staff behavior and how the game functions in general.

True, but what I'm trying to say is the game generates less than before and the bug that caused problems at 6 has been removed... and also caused less objectives. Can revert if the less objs is wanted, but I presumed otherwise.

**Comment by misandrie** [comment]
> > Steal objectives are more often than not kill objectives with extra steps due to average command staff behavior and how the game functions in general.
> 
> True, but what I'm trying to say is the game generates less than before and the bug that caused problems at 6 has been removed... and also caused less objectives. Can revert if the less objs is wanted, but I presumed otherwise.

Needs maintainer or third party opinion on this.

**Comment by packmore** [comment]
> I fixed a recent den fuck up that made tots have less objectives. old tot objs always overshot to above 5, rarely above 6 diff and they removed that. well that causes 2 objective tots. Best way to emulate the weight overshoot? increase the weight to where it would overshoot, 6.
> 
> Why did den not do this? _We all know why._

i think this should be moved to a different PR, personally. especially because this PR has no mention of the change in its description. it almost feels misleading having that be snuck into this PR.

**Comment by TooSillyFennec** [comment]
> > I fixed a recent den fuck up that made tots have less objectives. old tot objs always overshot to above 5, rarely above 6 diff and they removed that. well that causes 2 objective tots. Best way to emulate the weight overshoot? increase the weight to where it would overshoot, 6.
> > Why did den not do this? _We all know why._
> 
> i think this should be moved to a different PR, personally. especially because this PR has no mention of the change in its description. it almost feels misleading having that be snuck into this PR.

Yeah, honestly discussion is needed and this PR's not the place for that. Also, this was an attempt at reverting the _effect_ of the den change without reverting the bug fix. Personally though player selection might be the way to go, but again not the PR for that

**Comment by TooSillyFennec** [comment]
Made an issue for that type of thing, otherwise I'm probably not gonna touch that within this PR or the next few really.

**Review by Piras314 — APPROVED** [review]

**Pull request merged**